Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "shuttle"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
Let's do this in a separate commit so we can start writing a changelog too.

Reply:
Yeah, good point.  Resubmitting